src/move.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs/promises');
+const path = require('path');
+
+const moveFile = async(file, destination) => {
```

Comment:
I'm not sure, that it will work correctly if we'll pass just one argument, or none at all. Maybe it'll be better to foresee this and show an error in the console?

Reply:
You may be right, it isn't necessary in this task, but it would be nice to have